cfpb/consumerfinance.gov
# Carousel: Add text to image link

The carousel image is a link, but it only contained an image. This PR adds visually hidden text for screen readers.

## Additions

-  Add text to image link in carousel.

## How to test this PR

1. Pull branch and run site and generate a lighthouse report for the homepage. It should not flag the carousel image link.
